packages/client/components/ParabolScopingSearchResults.tsx

```diff
@@ -9,18 +10,28 @@ import useLoadMoreOnScrollBottom from '~/hooks/useLoadMoreOnScrollBottom'
 import IntegrationScopingNoResults from './IntegrationScopingNoResults'
 import useRecordIdsWithStages from '~/hooks/useRecordIdsWithStages'
 import {NewMeetingPhaseTypeEnum} from '~/types/graphql'
+import NewIntegrationRecordButton from './NewIntegrationRecordButton'
+import NewParabolTaskInput from './NewParabolTaskInput'
 
+const ResultScroller = styled('div')({
+  overflow: 'auto'
+})
 interface Props {
   relay: RelayPaginationProp
   viewer: ParabolScopingSearchResults_viewer | null
   meeting: ParabolScopingSearchResults_meeting
 }
 
+const StyledNewIntegrationRecordButton = styled(NewIntegrationRecordButton)({
+  left: '174%',
```

Comment:
numbers like this scare me when it comes to being mobile responsive (looks like i missed this on the last go-around for the jira one). 
is there a way you could position this (and the underlying NewIntegrationRecordButton) differently to avoid the percentages & get it on a 4px grid scale?

for example, make the Button a child of a `position: relative` parent that takes up the whole section & then add `bottom: 8, right: 8` or similar

Reply:
great point, changed it!